docs/concepts.md

```diff
@@ -67,6 +68,96 @@ that are specific to a particular Node (like per-process UNIX file descriptors),
 so handle value 3 for Node A identify a different channel than handle value 3
 for Node B (unless there happens to be a coincidence of numbering).
 
+## Labels
+
+Labels are used to enforce information flow control (IFC) between nodes and
+channels in an Oak Application and the outside world.
+
+Each node and each channel is associated with a label at creation time. The
+label does not change any more after that (apart from explicit declassification
+operations, which are not currently supported).
+
+A label has two components: **secrecy** and **integrity**.
+
+Each component is represented as an (unordered) set of tags, and each tag refers
```

Comment:
Consider cutting (unordered). I think it is unlikely that a reader will consider tags to have an authority ordering unless they are already familiar with the area. The text also uses order operators and implies that secrecy and integrity are ordered (e.g. with phrases like "more secret").

Reply:
I didn't meant that the authority does not have an order, just that the order of tags within each component is irrelevant. Technically this is implied by the fact that it is a set, but since in Rust it may end up being represented as a `Vec` (which has a specified order), I thought I may disambiguate upfront. But happy to remove this if it makes things confusing.

Reply:
Ah okay, given that users could think the implementation uses Vec, saying that they are unordered sets does seem useful :)